src/main/resources/messages/MyBundle.properties

```
@@ -1,2 +1,6 @@
 org.jqassistant.tooling.intellij.inlay.name = Report results
 org.jqassistant.tooling.intellij.inlay.description = Display inlay report results
+
+inspections.group.name = jQAssistant Support
+inspection.rule.xml.validity.display.name = Xml rule source validity
+id.contains.wildcard.characters = Value contains reserved wildcard characters
```

```suggestion
id.contains.wildcard.characters = Value contains reserved wildcard characters (*, &)
```
I think we should add the two characters to the message so that users know what characters are meant because the squiggly line is under the whole rule id instead of only the invalid character

---

I added a quick fix that removes the wildcard characters instead. I think this is better UX wise.